src/sentry/issues/event.schema.json

```diff
@@ -1810,7 +1810,7 @@
               ]
             }
           },
-          "additionalProperties": false
+          "additionalProperties": true
```

Comment:
It's kind of hard to see it in the diff, but it changes it in `definitions/Frame` which is referenced in `definitions/RawStacktrace`